apps/AEPSampleApp/package.json

```diff
@@ -18,23 +18,27 @@
     "@adobe/react-native-aepedgeidentity": "^1.0.0",
     "@adobe/react-native-aepmessaging": "^1.0.0-beta.2",
     "@adobe/react-native-aepuserprofile": "^1.0.0",
+    "@adobe/react-native-aepoptimize": "^1.0.0-alpha.1",
     "@react-native-community/masked-view": "^0.1.11",
-    "@react-navigation/bottom-tabs": "^5.8.0",
-    "@react-navigation/core": "^5.12.3",
-    "@react-navigation/drawer": "^5.9.0",
-    "@react-navigation/native": "^5.7.3",
-    "@react-navigation/routers": "^5.7.2",
-    "@react-navigation/stack": "^5.9.0",
-    "react": "17.0.1",
-    "react-native": "0.64.2",
-    "react-native-gesture-handler": "^1.10.3",
-    "react-native-reanimated": "^2.2.0",
+    "@react-navigation/bottom-tabs": "^6.3.1",
+    "@react-navigation/core": "^6.2.1",
+    "@react-navigation/drawer": "^6.4.1",
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/routers": "^6.1.0",
+    "@react-navigation/stack": "^6.2.1",
+    "react": "18.1.0",
+    "react-native": "0.68.2",
+    "react-native-gesture-handler": "^2.4.2",
+    "react-native-reanimated": "2.8.0",
```

Comment:
We need to keep reanimated version at 2.2.4 to avoid the error in the latest build.